pom.xml

```diff
@@ -26,8 +26,8 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/plugin-compat-tester</gitHubRepo>
-    <logbackVersion>1.4.5</logbackVersion>
```

Comment:
With the removal of Logback this is no longer needed and has therefore been deleted.